addons/web/static/src/js/tour.js

```diff
@@ -562,6 +563,16 @@ var Tour = {
         var $dropZone = $(".oe_drop_zone").first();
         var dropPosition = $dropZone.position();
         $dropZone.trigger($.Event("mouseup", { which: 1, pageX: dropPosition.left, pageY: dropPosition.top }));
+    },
+    scrollIntoView: function (element){
+        // if element is not on window (screen) then it scroll down to get popup visible
```

Comment:
Please, could you rename element in $element to indicate that it is a jQuery variable. (easier to debug)
Thanks